requirements.txt

```diff
@@ -1,11 +1,11 @@
-pip==10.0.1
+pip>=10.0.1
```

Comment:
Better to specify a particular version number here that is known to work. (These versions get used by travis-ci.) 

It's in setup.py where we specify a range of versions that our software should work with.

Reply:
I did that because on my computer pip 10.0.1 would not allow me to do things a more recent version (18.0 I think) did